package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "ssb-binary-field-encodings-spec",
+  "description": "Binary Field Encodings (BFE) spec for Secure Scuttlebutt (SSB)",
+  "version": "0.1.0",
+  "homepage": "https://github.com/ssb-ngi-pointer/ssb-binary-field-encodings-spec",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/ssb-ngi-pointer/ssb-binary-field-encodings-spec.git"
+  },
+  "main": "bfe.json",
+  "scripts": {
+    "test": "tape *.test.js | tap-spec"
+  },
+  "author": "Anders Rune Jensen <arj03@protonmail.ch>",
+  "contributors": [
+    "Andre Staltz <contact@staltz.com>",
+    "Mix Irving <mix@protozoa.nz>"
+  ],
+  "license": "MIT",
```

Comment:
By the way, since this module is primarily data, not code, I recommend a data license such as what we did for ssb-caps: https://github.com/ssbc/ssb-caps/blob/838581c6bc47fab88b241c74c537798857d38c11/package.json#L10-L11

Reply:
Works for me

Reply:
Please go ahead and check it @staltz

Reply:
executed this change